flasc/energy_ratio/energy_ratio.py

```diff
@@ -297,6 +316,10 @@ def compute_energy_ratio(er_in: EnergyRatioInput,
         weight_by (str): How to weight the energy ratio, options are 'min', , or 'sum'.  'min' means
             the minimum count across the dataframes is used to weight the energy ratio.   'sum' means the sum of the counts
             across the dataframes is used to weight the energy ratio.
+        df_freq (pd.Dataframe): A dataframe which specifies the frequency of the ws/wd bin combinations.  Provides
+            a method to used an explicit or long-term weigthing of various bins.  Dataframe should include
+            columns ws, wd and freq_val.  ws and wd should correspond to the bin centers resulting from
+            the choices of the ws/wd_min / _max / _step.  Defatuls to None
```

Comment:
Suggest describing how the weights are handled when the df_freq bins don't match the bins given by ws/wd min, max, step perfectly. So if df_freq has extra bins that aren't included in those given by ws/wd min, max, step, they will be ignored in the energy ratio calculation. And any bins given by ws/wd min, max, step not present in df_freq will be assigned a frequency of zero. At least I think that's how those situations are handled.